smart_contracts.rst

```diff
@@ -275,23 +305,23 @@ Unlocks all pending transfers by providing the entire merkle tree of pending tra
 ::
 
     function unlock(
-        uint channel_identifier,
         address participant,
         address partner,
-        bytes merkle_tree)
+        bytes merkle_tree_leaves
+    )
         public
 
 ::
 
     event ChannelUnlocked(uint256 channel_identifier, address participant, uint256 unlocked_amount, uint256 returned_tokens);
 
-- ``channel_identifier``: Channel identifier assigned by the current contract.
 - ``participant``: Ethereum address of the channel participant who will receive the unlocked tokens that correspond to the pending transfers that have a revealed secret.
 - ``partner``: Ethereum address of the channel participant that pays the amount of tokens that correspond to the pending transfers that have a revealed secret. This address will receive the rest of the tokens that correspond to the pending transfers that have not finalized and do not have a revelead secret.
-- ``merkle_tree``: The entire merkle tree of pending transfers. It contains tightly packed data for each transfer, consisting of ``expiration_block``, ``locked_amount``, ``secrethash``.
+- ``merkle_tree_leaves``: The entire merkle tree of pending transfers. It contains tightly packed data for each transfer, consisting of ``expiration_block``, ``locked_amount``, ``secrethash``.
```

Comment:
I must have missed this. Why specify in the variable names that it's the leaves?

Reply:
Then perhaps also change the description here since saying: "The entire merkle tree" gives a different expectation. Saying the data of the merkle tree as you described in your comment would be more accurate